ExcelExample-FrenchTraffic/model_definition.yaml

```diff
@@ -1,9 +1,11 @@
 kind: M
-api_version: v1beta2
+api_version: v1beta3
 metadata:
   display_name: MS Excel Win10 Test
   name: kjs-test-excel
   publisher: KJS Test
+  contact_point_name:  KJS Test
```

Comment:
Can I just double check that this was intentionally set to `KJS Test` rather than DAFNI?

Reply:
I think it's better to set it to the standard DAFNI - (KJS Test, is just my initials !)